projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/remote_data_job_executor.py

```diff
@@ -10,9 +11,11 @@
 
 
 class RemoteDataJobExecutor(IDataJobExecutor):
-    def start_job(self, job_name: str, team_name: str):
+    def start_job(self, job_name: str, team_name: str, arguments: Dict = None):
```

Comment:
nit: this seems shady - I can recall there is IJobArguments interface, and there should be usages within vdk-core since all data jobs have arguments. Why does not this apply to dags..?

Reply:
Well, I guess I missed it. Thanks! Applied it now.